shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -225,17 +235,20 @@ public void ensureInitializationComplete(
         shellArgs.add("--log-tag=" + settings.getLogTag());
       }
 
+      ActivityManager activityManager =
+          (ActivityManager) applicationContext.getSystemService(Context.ACTIVITY_SERVICE);
+      int oldGenHeapSizeMegaBytes = activityManager.getLargeMemoryClass();
```

Comment:
I don't think we actually set largeHeap="true" in our AndroidManifest template right? Should we?

Reply:
That's for the Java heap. This property is unaffected by that setting - it's what the Java heap would be set to if largeHeap was set to true.

I don't think we need largeHeap set to true, since we really don't do much with the Java heap.

Reply:
ah got it. Thanks